usr/share/rear/conf/default.conf

```diff
@@ -1063,6 +1063,13 @@ USB_BOOTLOADER=
 # if an USB device was manually mounted to avoid recursive backups:
 AUTOEXCLUDE_USB_PATH=()
 
+# USB EFI booting can benifit with a better search string then the default:
+# GRUB2_SET_USB_ROOT="search --no-floppy --set=root --label REAR-EFI
+# as hardcoded in script output/USB/Linux-i386/100_create_efiboot.sh
+# Only to be used by experts. An example of a different setup could be:
+# GRUB2_SET_USB_ROOT="search --no-floppy --set=root --label REAR-EFI --hint hd0,msdos1"
+GRUB2_SET_USB_ROOT=""
```

Comment:
We have already some other `GRUB2_...` config variables
at another place in default.conf - currently starting
at line 3182 with GRUB2_INSTALL_DEVICES
https://github.com/rear/rear/blob/master/usr/share/rear/conf/default.conf#L3182
which is for installing GRUB2 on the target system during "rear recover"
and then other `GRUB2_...` config variables but with different meaning
GRUB2_MODULES_UEFI_LOAD
GRUB2_MODULES_UEFI
GRUB2_DEFAULT_BOOT
GRUB2_TIMEOUT
because those are not for installing GRUB2
on the recreated target system during "rear recover"
but when using GRUB2 as ReaR recovery system bootloader
during "rear mkrescue".

So I wonder if the current places in default.conf
where we describe `GRUB2_...` config variables are OK.